package.json

```diff
@@ -36,12 +34,10 @@
         "build": "npm-run-all build:targets gen:declarations",
         "build:targets": "gulp build",
         "prebuild:targets": "rm -rf dist && rm -rf lib",
-        "tsc": "tsc",
+        "postbuild": "bundlesize",
```

Comment:
I think having it here as `postbuild` actually makes sense. Cuz it's not actually a "build" step, but something you wanna do afterwards 🎉

Reply:
I agree, but it was failing for me. I may have to review the docs on it to know how it works.